### ISSUE TITLE
Use a red bar instead of the dot

### ISSUE THREAD

**Issue by adrianotrentim** [afriscic/BarcodeScanning.Native.Maui#30]
It would be cool if we could use a red bar instead of the dot. A thin bar, which could be even better than the dot, filling the entire reading area, to help identify the beginning and end of the barcode reading

**Comment by afriscic** [comment]
Hello.
Do you mean just some line (horizontal)?

**Comment by adrianotrentim** [comment]
Hi.

Yes, a horizontal red bar, to help show the user where to focus, instead of the red dot

**Comment by afriscic** [comment]
I agree with @Ghostbird that using a bar is a bit archaic especially in context of 2D codes. But I also acknowledge that for some people are more used to old barcode scanners that displayed a red bar when scanning. As all aiming features are optional you can easily enable or disable whatever suits you.
No ETA for now tho...

**Comment by Ghostbird** [comment]
@afriscic I agree with your conclusion, thought you seem to have misunderstood my reason.
My point was that there's no inherent relation between the rotational of the device, and the target barcode. In the face of an auto-rotate option that keeps the line horizontal (or at least prevents verticality) it can make certain real-world uses impossible or impractical. This harms the user experience, especially when there's no reason to view the barcode under a certain rotation to an inherently 2D camera scanner.

Summarised: The horizontal line is a fake throwback to historic limitations of 1D scanners, that confuses the UX.

**Comment by Ghostbird** [comment]
@afriscic I agree with your conclusion, thought you seem to have misunderstood my reason.

My point was that there's no inherent relation between the rotation of the device, and the target barcode. The horizontal line is a fake throwback to historic limitations of 1D scanners, that confuses the UX. To the user it evokes a limitation that they must work to overcome, even though there actually is no such limitation
The addional actions it may make the user perform might be harmful to the user (RSI), device (unnescessary maipulations increase chance of drop/hitting things) and functioning of the software (harder to scan while the user is in the progress of unnecessary device manipulations).
